fuel-core-interfaces/src/model.rs

```diff
@@ -9,12 +9,16 @@ pub use block::{FuelBlock, FuelBlockConsensus, FuelBlockDb, FuelBlockHeader, Sea
 pub use block_height::BlockHeight;
 pub use coin::{Coin, CoinStatus};
 pub use deposit_coin::DepositCoin;
+use fuel_types::Address;
 pub use txpool::{ArcTx, TxInfo};
 pub use vote::Vote;
 
 pub type DaBlockHeight = u32;
 pub type ValidatorStake = u64;
 
+pub type ValidatorAddress = Address;
+pub type ConsensusPublicKey = Address;
```

Comment:
An address is the hash of a public key. In this case, is `ConsensusPublicKey` actually a public key (compressed)? If so, it might be better to have it have a different _direct_ underlying type, even though the ultimate representation is still 32 bytes.

Reply:
ValidatorAddress is eth related address.
But for `ConsensusPublicKey`, my first thought is that this must be a public key as we are using it for verification, but as we can do recovery of the public key from signature, `ConsensusPublicKey` can be made as a hash. Maybe leave it as a compressed public key without prefix for now. In future, it could be a consensus change to make it into hash.

Reply:
For this pull request i just changed `Adress` to `Bytes32`